server/world/chunk/chunk.go

```diff
@@ -14,6 +14,8 @@ type Chunk struct {
 	r cube.Range
 	// air is the runtime ID of air.
 	air uint32
+	// modified is true if the chunk has been modified since it was last saved.
+	modified bool
```

Comment:
I think it'd make more sense to put this in the `chunkData` of the world package.